### PR TITLE
Added link to react-native guide

### PR DESCRIPTION
**Pull request by osamaq**
Wrote a step-by-step guide based on my experience as a complete beginner with fastlane and App Center.

### CODE REVIEW

**Comment by osamaq** [comment]
I signed it. Thanks!

File: docs/getting-started/cross-platform/react-native.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Most of the _fastlane_ docs on this page apply to React Native projects as well.
 As there is no official getting started guide yet, here are some publications by the _fastlane_ community:
 
 - [Shipping React Native apps with fastlane](https://carloscuesta.me/blog/shipping-react-native-apps-with-fastlane/)
+- [React Native, fastlane and App Center](https://github.com/osamaq/reactnative-fastlane-appcenter)
```
Comment (janpio) [inline docs/getting-started/cross-platform/react-native.md]: I think this gives some more context:
```suggestion
- [React Native, fastlane and Visual Studio App Center](https://github.com/osamaq/reactnative-fastlane-appcenter)
```

Comment (osamaq) [reply docs/getting-started/cross-platform/react-native.md]: Looks good! No objections.

**Comment by osamaq** [comment]
I am not sure what needs to be done 🤨. I am ok googlebot.

**Comment by janpio** [comment]
Nothing, googlebot can't handle the "Suggest changes" feature of GitHub yet.

Thanks for the link! 🚀

**Comment by osamaq** [comment]
Haha. I've never seen something like it. I am beginner and was preplexed. Thank you for the help!

**Pull request merged**